tests/unit/models/gpflux/test_sampler.py

```diff
@@ -63,6 +63,8 @@ def test_dgp_reparam_sampler_raises_for_invalid_sample_size(sample_size: int) ->
         DeepGaussianProcessReparamSampler(sample_size, dgp)
 
 
+"""
+
```

Comment:
you can do instead `@pytest.mark.skip` just above the function

Reply:
thanks.